### PR TITLE
Add order access to cargo technicians

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Don't merge the PR before all the maps have public cargo request consoles.
image

## Why / Balance
image

## Technical details
YAML change.

## Media

Not needed.

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Hopefully none.

**Changelog**
:cl:
- tweak: Now cargo technicians can approve orders.

### CODE REVIEW

**Comment by deltanedas** [comment]
LO back to being a fake job its over

**Comment by Kr8art** [comment]
Can't wait for the clown to steal order access from a random SSD cargo technician and buy twenty foam toy crates on the public request console.

**Comment by deltanedas** [comment]
if you steal an id you can just walk 2 meters inside cargo

**Review by Colin-Tel — APPROVED** [review]
Most maps now have public-facing cargo request computers.

**Review by deltanedas — APPROVED** [review]

**Comment by Lyndomen** [comment]
Just as a heads up, Direction is gonna be checking this one out extra close, as we (I) am very nervous about it.

**Comment by JustTNE** [comment]
> if you steal an id you can just walk 2 meters inside cargo

I think the argument here was that cargo techs having order access makes it easy for someone to self antag with that order access, regardless of location of the console.

**Comment by deltanedas** [comment]
you can easily selfantag by breaking into LOs room to get the spare id
and if all of cargo sees a selfantag buying guns or whatever theyll just bonk with their bats

**Comment by CopperCrow9** [comment]
So I will share my experience with this from a Logistics Officers perspective: while this is nice for the Technicians, it’s way to chaotic for Cargo as a whole. Money usually is wasted on a lot of stuff that the station doesn’t need, especially when we’re low on funds already. 

Stocks are also included with orders access. I will usually put money on high end stocks and find someone sold it with, at best, little net gain. 

Cargo, while being a chaotic environment, is amplified by this. I had a shift where someone ordered 10k worth of instruments just to make a single DAW. There was also a lot of instances where our money and resources were put into Epistemics even though I said not to because there was no gain coming from them.

TLDR: Cargo Techs are having a blast, but the Logistics Officer is now reduced to a Cargo Tech with an IPad while going insane because they can’t regulate their department.
